File: RepoTransBench/target_projects/Java/C#/adennie_fb-android-dagger/tests/original/InjectingActivityModuleTest.cs
using Moq;
using Xunit;
using ProjectName;

namespace OriginalTests
{
    public class InjectingActivityModuleTest
    {
        private Mock<Activity> _mockActivity;
        private Mock<Injector> _mockInjector;
        private InjectingActivityModule _module;

        public InjectingActivityModuleTest()
        {
            _mockActivity = new Mock<Activity>();
            _mockInjector = new Mock<Injector>();
            _module = new InjectingActivityModule(_mockActivity.Object, _mockInjector.Object);
        }

        [Fact]
        public void TestProvideActivityContext()
        {
            var context = _module.ProvideActivityContext();
            Assert.NotNull(context);
            Assert.Equal(_mockActivity.Object, context);
        }

        [Fact]
        public void TestProvideActivity()
        {
            var activity = _module.ProvideActivity();
            Assert.NotNull(activity);
            Assert.Equal(_mockActivity.Object, activity);
        }

        [Fact]
        public void TestProvideActivityInjector()
        {
            var injector = _module.ProvideActivityInjector();
            Assert.NotNull(injector);
            Assert.Equal(_mockInjector.Object, injector);
        }
    }
}

File: RepoTransBench/source_projects/Java/adennie_fb-android-dagger/src/test/java/com/fizzbuzz/android/dagger/InjectingActivityModuleTest.java
package com.fizzbuzz.android.dagger;

import android.app.Activity;
import android.content.Context;
import org.junit.Before;
import org.junit.Test;
import org.mockito.Mock;
import org.mockito.MockitoAnnotations;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertNotNull;
import static org.mockito.Mockito.mock;

public class InjectingActivityModuleTest {

    @Mock
    private Activity mockActivity;
    @Mock
    private Injector mockInjector;

    private InjectingActivityModule module;

    @Before
    public void setUp() {
        MockitoAnnotations.initMocks(this);
        module = new InjectingActivityModule(mockActivity, mockInjector);
    }

    @Test
    public void testProvideActivityContext() {
        Context providedContext = module.provideActivityContext();
        assertNotNull(providedContext);
        assertEquals(mockActivity, providedContext);
    }

    @Test
    public void testProvideActivity() {
        Activity providedActivity = module.provideActivity();
        assertNotNull(providedActivity);
        assertEquals(mockActivity, providedActivity);
    }

    @Test
    public void testProvideActivityInjector() {
        Injector providedInjector = module.provideActivityInjector();
        assertNotNull(providedInjector);
        assertEquals(mockInjector, providedInjector);
    }
}